webpack.config.babel.js

```diff
@@ -0,0 +1,75 @@
+import path from 'path';
+import webpack from 'webpack';
+
+let config = {
+	admin: path.resolve(__dirname, './admin/client/index'),
+	entry: {
+		packages: [
+			'async',
+			'blacklist',
+			'bytes',
+			'classnames',
+			'color',
+			'display-name',
+			'elemental',
+			'expression-match',
+			'history',
+			'i',
+			'list-to-array',
+			'marked',
+			'moment',
+			'numeral',
+			'qs',
+			'react-addons-css-transition-group',
+			'react-alt-text',
+			'react-color',
+			'react-day-picker',
+			'react-dnd',
+			'react-dnd-html5-backend',
+			'react-dom',
+			'react-select',
+			'react',
+			'store-prototype',
+			'vkey',
+			'xhr'
+		]
+	},
+	output: {
+		filename: '[name].js',
+		path: path.resolve(__dirname, './admin/public/js')
+	},
+	module: {
+		loaders: [
+			{ test: /\.jsx?$/, loader: 'babel', exclude: /node_modules/ }
+		]
+	},
+	plugins: [
+		new webpack.DefinePlugin({
+			ENV: process.env.NODE_ENV
```

Comment:
Is this correct? I was under the impression it had to be `{process:{env:{NODE_ENV: JSON.stringify(process.env.NODE_ENV)}}}`